docs/packages/pkg/PROJ4.rst

```diff
@@ -0,0 +1,19 @@
+.. spelling::
+
+    PROJ4
+
+.. index:: GIS ; PROJ4
+
+.. _pkg.PROJ4:
+
+proj4
+===
+
+-  `Official <https://github.com/OSGeo/proj.4>`__
+-  `Hunterized <https://github.com/hunter-packages/PROJ4>`__
```

Comment:
There is no hunterization fork.

Reply:
How am i supposed to create one? Maybe i missed it in the tutorial:/ Or is this only for altered sources?

Reply:
If you're using upstream sources there is no need to create a fork:
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#cmake-no-dependencies

Simply remove this line from documentation.